opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -154,12 +155,17 @@ def __init__(
         metric_readers: Sequence[MetricReader] = (),
         resource: Resource = Resource.create({}),
         shutdown_on_exit: bool = True,
+        views: Sequence[View] = (),
+        enable_default_view: bool = True,
     ):
         self._lock = Lock()
         self._meter_lock = Lock()
         self._atexit_handler = None
         self._sdk_config = SdkConfiguration(
-            resource=resource, metric_readers=metric_readers
+            resource=resource,
+            metric_readers=metric_readers,
+            views=views,
```

Comment:
this becomes stale since you modify the list below

Reply:
Fixed